### PR TITLE
[Test Structure] Populate emibSampleTest

### PR DESCRIPTION
**Pull request by MichaelCherry**
# Description

Script to populate the emibSampleTest test model

These values will be used to look up the sample test using the string "emibSampleTest", as int pks may be reliable across environments.

This uses the model created in PR #315 

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

image
Note: my item_id is 77; your's will likely be 1

# Testing

N/A (can look in the postgres table for the above values)

# Checklist

Applicable for all code changes.

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new compiler warnings
~~- [ ] My changes generate no new accessibility errors and/or warnings~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)~~
~~- [ ] My changes look good on IE 11+ and Chrome~~

### CODE REVIEW

File: backend/custom_models/migrations/0007_upload_emib_sample_test.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.1.7 on 2019-06-03 15:27
+
```
Comment (caleybrock) [inline backend/custom_models/migrations/0007_upload_emib_sample_test.py]: Can you add a comment saying you made this - co it's clear its not one of the autogenerated ones.

File: backend/custom_models/migrations/0007_upload_emib_sample_test.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+def upload_emib_sample(apps, schema_editor):
+    # get models
+    language = apps.get_model("custom_models", "Language")
+    item_text = apps.get_model("custom_models", "ItemText")
+    test = apps.get_model("custom_models", "Test")
+    # get db alias
+    db_alias = schema_editor.connection.alias
+
+    # get emib_sample item
+    l_english = language.objects.using(db_alias).filter(
```
Comment (caleybrock) [inline backend/custom_models/migrations/0007_upload_emib_sample_test.py]: how do you connect the french with this test as well?

File: backend/custom_models/migrations/0007_upload_emib_sample_test.py (hunk 1 above)
Comment (MichaelCherry) [reply backend/custom_models/migrations/0007_upload_emib_sample_test.py]: fixed in 137f4ab

File: backend/custom_models/migrations/0007_upload_emib_sample_test.py (hunk 2 above)
Comment (MichaelCherry) [reply backend/custom_models/migrations/0007_upload_emib_sample_test.py]: I'm actually connecting the item. But I'm finding the item from the english item_text. I could just as easily use the french. I use "emib_sample_item_text_en.**item_id**" which is the item not "emib_sample_item_text_en" which is the item_text.

I don't want to rely on the emib sample being the **only** item of type test, or even the first item of type test, thus why I look up the item_text.

The en/fr item_texts are already connected to the item.

**Pull request merged**